plugins/modules/icinga2_host.py

```diff
@@ -256,9 +256,8 @@ def main():
     state = module.params["state"]
     name = module.params["name"]
     zone = module.params["zone"]
-    template = [name]
```

Comment:
The variable `template` is now no longer initialized if the module's `template` parameter isn't used.

Reply:
Indeed. Fixed it in a new commit.